WalletWasabi.Fluent/Controls/ContentArea.axaml.cs

```diff
@@ -46,9 +46,23 @@ public class ContentArea : ContentControl
 	public static readonly StyledProperty<IBrush> HeaderBackgroundProperty =
 		AvaloniaProperty.Register<ContentArea, IBrush>(nameof(HeaderBackground));
 
-	private IContentPresenter? _titlePresenter;
+	public static readonly DirectProperty<ContentArea, object?> ButtonAreaContentProperty =
+		AvaloniaProperty.RegisterDirect<ContentArea, object?>(
+			"ButtonAreaContent",
+			o => o.ButtonAreaContent,
+			(o, v) => o.ButtonAreaContent = v);
+
+	private object? _buttonAreaContent;
 	private IContentPresenter? _captionPresenter;
 
+	private IContentPresenter? _titlePresenter;
+
+	public object? ButtonAreaContent
+	{
+		get => _buttonAreaContent;
+		set => SetAndRaise(ButtonAreaContentProperty, ref _buttonAreaContent, value);
```

Comment:
My mistake! I created it incorrectly as Direct Property instead of StyledProperty (https://docs.avaloniaui.net/docs/authoring-controls/defining-properties#readonly-properties). Anyways, the changes in this control are no longer needed. I will restore to the previous version.